scripts/ci-install-ubuntu-latest.sh

```diff
@@ -1,5 +1,8 @@
 sudo apt-get update
 sudo add-apt-repository universe
+wget -O - https://github.com/tree-sitter/tree-sitter/releases/download/v0.19.4/tree-sitter-linux-x64.gz | gunzip -c > tree-sitter
+sudo cp ./tree-sitter /usr/bin/tree-sitter
+sudo chmod uog+rwx /usr/bin/tree-sitter
```

Comment:
:+1: 

Very strange though...
` Warning: parser for elixir is not installed`

Reply:
It looks like the script thinks `npm` is required:

> Skipping language elixir on CI (requires npm)!

I haven't looked into why that is yet. I will take a look tomorrow.

Reply:
That's the error message right after this check. But it does not fit very well to the check.